indexer/postgres/conn.go

```diff
@@ -5,8 +5,8 @@ import (
 	"database/sql"
 )
 
-// DBConn is an interface that abstracts the *sql.DB, *sql.Tx and *sql.Conn types.
-type DBConn interface {
+// dbConn is an interface that abstracts the *sql.DB, *sql.Tx and *sql.Conn types.
+type dbConn interface {
```

Comment:
Yes all the functions using this interface are now private. Previously I made a lot of methods and types public because I needed them for testing. But I found another way to test so I don't need them to be public and think it's better to keep the public API small for now.

Reply:
Yes. I can make all the methods explicitly private if that would be less confusing

Reply:
Updated in https://github.com/cosmos/cosmos-sdk/pull/21363/commits/c73f7a262eac466d3f6e08711118cbf3bcdd6b99